tests/load/options.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package load
+
+import (
+	"context"
+	"time"
+
+	"github.com/ava-labs/libevm/core/types"
+
+	"github.com/ava-labs/avalanchego/ids"
+)
+
+// IssuanceReceipt is the information known after issuing a transaction.
+type IssuanceReceipt struct {
```

Comment:
1. I think we can probably get rid of these receipt types if we get rid of the options type
2. I don't think we need them anyways, since we can just pass in the fields they contain as function params anyways